src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs

```diff
@@ -85,8 +92,70 @@ public Test()
             public string? EditorConfig { get; set; }
 
 #if !CODE_STYLE
+            private readonly List<AdhocWorkspace> _workspaces = new();
+
             protected override AnalyzerOptions GetAnalyzerOptions(Project project)
                 => new WorkspaceAnalyzerOptions(base.GetAnalyzerOptions(project), project.Solution);
+
+            /// <summary>
+            /// The <see cref="TestHost"/> we want this test to run in.  Defaults to <see cref="TestHost.InProcess"/> if unspecified.
+            /// </summary>
+            public TestHost TestHost { get; set; } = TestHost.InProcess;
```

Comment:
Not sure if it's worth testing both in process and out-of-proc.

Reply:
it is :)   it catches real issues.

Reply:
(in this case, the feature has a large oop piece of work that it does, so we really want to validate that that is working properly).

Reply:
@CyrusNajmabadi Thanks for clarifying